doc/api/util.md

```diff
@@ -271,6 +271,25 @@ util.inspect(obj);
   // "{ bar: 'baz' }"
 ```
 
+### util.inspect.defaultOptions
+
+The `defaultOptions` property allows customization of the default options used
+by `util.inspect`. This is useful for functions like `console.log` or
+`util.format` which implicitly call into `util.inspect`. The property shall be
+set to an object containing one or more valid [`util.inspect()`][] options.
+Setting properties directly is also supported.
+
+Returns the current default options as an object.
```

Comment:
It's not a function, so it doesn't "return" anything, right? I would just remove this line.